packages/core/admin/admin/src/components/GuidedTour/Modal/components/Stepper.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { pxToRem } from '@strapi/helper-plugin';
+import { Typography } from '@strapi/design-system/Typography';
+import { Button } from '@strapi/design-system/Button';
+import { LinkButton } from '@strapi/design-system/LinkButton';
+import { Box } from '@strapi/design-system/Box';
+import { Flex } from '@strapi/design-system/Flex';
+import ArrowRight from '@strapi/icons/ArrowRight';
+import Content from './Content';
+import StepLine from '../../Stepper/StepLine';
+import StepNumberWithPadding from './StepNumberWithPadding';
+import { IS_DONE, IS_ACTIVE } from '../../constants';
+
+const StepperModal = ({
+  title,
+  content,
+  cta,
+  onCtaClick,
+  sectionIndex,
+  stepIndex,
+  hasSectionAfter,
+}) => {
+  const { formatMessage } = useIntl();
+
+  const hasSectionBefore = sectionIndex > 0;
+  const hasStepsBefore = stepIndex > 0;
+  const nextSectionIndex = sectionIndex + 1;
+
+  return (
+    <>
+      <Flex alignItems="stretch">
+        <Flex marginRight={8} justifyContent="center" minWidth={pxToRem(30)}>
+          {hasSectionBefore && <StepLine type={IS_DONE} minHeight={pxToRem(24)} />}
+        </Flex>
+        <Typography variant="sigma" textColor="primary600">
+          {formatMessage({
+            id: 'app.components.GuidedTour.title',
+            defaultMessage: '3 simple step',
```

Comment:
```suggestion
            defaultMessage: '3 simple steps',
```

Same feedback as above about the choice of words though.